server/system/aws.js

```diff
@@ -0,0 +1,36 @@
+const aws = require('aws-sdk')
+
+const inviteUser = email =>
+  new Promise((resolve, reject) => {
+    aws.config.update({
```

Comment:
is aws.config needed at each inverUser invocation? or can it be done just once?

Reply:
We might want to rotate them in the future but for now setting them just once

Reply:
where is the config happening now? I couldn't see it..